tests/Unit/PointwiseFunctions/Elasticity/Test_Strain.cpp

```diff
@@ -0,0 +1,113 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <cstddef>
+#include <string>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/CoordinateMaps/Affine.hpp"
+#include "Domain/CoordinateMaps/CoordinateMap.hpp"
+#include "Domain/CoordinateMaps/ProductMaps.hpp"
+#include "Domain/CoordinateMaps/ProductMaps.tpp"
+#include "Domain/LogicalCoordinates.hpp"
+#include "Elliptic/Systems/Elasticity/Tags.hpp"
+#include "Helpers/DataStructures/DataBox/TestHelpers.hpp"
+#include "Helpers/DataStructures/MakeWithRandomValues.hpp"
+#include "PointwiseFunctions/Elasticity/Strain.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace {
+
+// Polynomial functions of max. degree 2, so the differentiation is exact on 3
+// grid points
+template <size_t Dim>
+tnsr::I<DataVector, Dim> polynomial_displacement(
+    const tnsr::I<DataVector, Dim>& x) {
+  tnsr::I<DataVector, Dim> displacement{x.begin()->size()};
+  get<0>(displacement) = square(get<0>(x)) + 2. * get<1>(x);
+  get<1>(displacement) = square(get<1>(x)) + 3. * get<0>(x);
+  if constexpr (Dim == 3) {
+    get<2>(displacement) = square(get<2>(x)) + get<0>(x) + 4. * get<1>(x);
+  }
+  return displacement;
+}
+
+template <size_t Dim>
```

Comment:
a short comment that this is $\nabla_{(i} \xi_{j)}$ of polynomial_displacement might be helpful